src/main/java/com/wiley/autotest/selenium/extensions/internal/SelectImpl.java

```diff
@@ -164,20 +166,26 @@ public String getSelectedValue() {
     @Override
     public List<TeasyElement> getOptions() {
         try {
-            return new TransformedWebElementList(wrappedSelect().getOptions()).toOurWebElementList();
+            return wrappedSelect().getOptions()
+                    .stream()
+                    .map(webElement -> new DomTeasyElement(new TeasyElementData(getWrappedElement(), By.tagName("option"))))
+                    .collect(Collectors.toList());
         } catch (UndeclaredThrowableException ignored) {
             //TODO VF fix it in other places
             //Sometimes this test fails in ie due to such exception
             TestUtils.waitForSomeTime(3000, EXPLANATION_MESSAGE_FOR_WAIT);
-            return new TransformedWebElementList(wrappedSelect().getOptions()).toOurWebElementList();
+            return wrappedSelect().getOptions()
```

Comment:
decided to keep this duplicates for now, as we need to completely rework our custom elements approach...